packages/langium/src/validation/document-validator.ts

```diff
@@ -232,13 +233,13 @@ export function getDiagnosticRange<N extends AstNode>(info: DiagnosticInfo<N, st
 export function toDiagnosticSeverity(severity: 'error' | 'warning' | 'info' | 'hint'): DiagnosticSeverity {
     switch (severity) {
         case 'error':
-            return DiagnosticSeverity.Error;
+            return 1; // according to vscode-languageserver-types/lib/esm/main.js#DiagnosticSeverity.Error
```

Comment:
I now replaced the constants like `DiagnosticSeverity.Information` from `vscode-languageserver-types` by their values, and left an info on their original definition.

Reply:
Not a big fan of this - users that don't want to include `vscode-languageserver-types` in their build, are now only allowed to use the numbers themselves (or have to redeclare the enum themselves). We should offer the enum ourselves.

Reply:
Actually this peace of code is only used to convert the Langium notion of `severity` being defined as `'error' | 'warning' | 'info' | 'hint'` into the LSP notion while composing LSP diagnostics, right?
Our code is just converting the values according to the protocol definition, and therefore I don't see why Langium should export copies of those definitions.
Do you have concrete use cases in mind? Why should users care about those constants?

Reply:
Ah, right I forgot. Alright, that's fine then 👍